src/OpenTelemetry.AutoInstrumentation/Configuration/ConfigurationKeys.cs

```diff
@@ -144,6 +145,40 @@ public static class Metrics
         public const string AdditionalSources = "OTEL_DOTNET_AUTO_METRICS_ADDITIONAL_SOURCES";
     }
 
+    /// <summary>
+    /// Configuration keys for logs exporter
+    /// </summary>
+    public static class Log
+    {
+        /// <summary>
+        /// Configuration key for the logs exporter to be used.
+        /// Default is <c>"otlp"</c>.
+        /// </summary>
+        public const string Exporter = "OTEL_LOGS_EXPORTER";
+
+        /// <summary>
+        /// Configuration key for whether the logs console exporter is enabled.
+        /// </summary>
+        public const string ConsoleExporterEnabled = "OTEL_DOTNET_AUTO_LOGS_CONSOLE_EXPORTER_ENABLED";
```

Comment:
btw, we need a follow up to add the new env vars to the proper markdown file.

Reply:
#1136

Reply:
I called out in the PR description, before the merge need to create a set of issues.

**TODO**: Tests, ChangeLog and Documentation through follow up PR. Will create new issues to track before merging the PR.
Creating Plugins are tricker and will get the covered once all other TODOs are complete.